pysat/coords.py

```diff
@@ -0,0 +1,263 @@
+import scipy
+import scipy.integrate
+import numpy as np
+
+
+def geocentric_to_ecef(latitude, longitude, altitude):
+    """Convert geocentric coordinates into ECEF
+    
+    Parameters
+    ----------
+    latitude : float or array_like
+        Geocentric latitude (degrees)
+    longitude : float or array_like
+        Geocentric longitude (degrees)
+    altitude : float or array_like
+        Height (km) above presumed spherical Earth with radius 6371 km.
+        
+    Returns
+    -------
+    x, y, z
+        numpy arrays of x, y, z locations in km
+        
+    """
+
+    r = 6371. + altitude
```

Comment:
Might be more elegant to define the earth radius as a variable at the top and give a reason for choosing this particular definition of the radius.

Reply:
Done.